addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1179,6 +1178,115 @@ QUnit.test('open 2 chat windows: check shift operations are available', async fu
     );
 });
 
+QUnit.test('open & fold 2 chat windows: check shift operations are available', async function (assert) {
```

Comment:
If we specify that, would be good to explicitly set 'LTR' in env, so would need this:
https://github.com/odoo/odoo/pull/54632#discussion_r465553535

Reply:
Not sure to understand how LTR or RTL could influence that test. Maybe with the new approach it's different, could you check ?

Reply:
Check the doc on the `visibleIndex` field. It's value in regard to left-right mean something different depending on text direction.

Reply:
Note that another design approach to this problem could be to transform the "shift left"/"shift right" into "shift previous"/"shift next" similar to tab cycling.

Now that I think about it, the current code for shifting is actually wrong. If you click left on RTL, I'm pretty sure it's going to go right in current state instead of left. (might need to check with at least 3 windows and use the one in the middle)

Reply:
> Check the doc on the `visibleIndex` field. Its value in regard to left-right mean something different depending on text direction.

Maybe but this test don't care, he just shift left the only one that can be left shifted. And the same with right. Should it be the first or the second, the left or the right one, this test just don't care

Reply:
> Now that I think about it, the current code for shifting is actually wrong. If you click left on RTL, I'm pretty sure it's going to go right in current state instead of left. (might need to check with at least 3 windows and use the one in the middle)

Maybe, I don't know, I think this is not the scope of the PR, the idea here is to check that even if folded, it can be shifted. If the shifting is not correct, then we need to dig into the shifting functionality.

Reply:
> Now that I think about it, the current code for shifting is actually wrong. If you click left on RTL, I'm pretty sure it's going to go right in current state instead of left. (might need to check with at least 3 windows and use the one in the middle)

Just checked and in RTL it's also working correctly

Reply:
It works, but the texts are wrong "shift right" for left shift, etc.
That's because the code in model still reason in terms of left/right shift, while it should be previous/next now with visibility order. Text on the shift buttons should use "left"/"right" wording for clarity instead of "next"/"previous".

Reply:
FYI task-2314205